okapi-core/src/main/java/org/folio/okapi/bean/TenantModuleDescriptor.java

```diff
@@ -58,4 +66,27 @@ public String getMessage() {
   public void setMessage(String message) {
     this.message = message;
   }
+
+  public Stage getStage() {
+    return stage;
+  }
+
+  public void setStage(Stage stage) {
+    this.stage = stage;
+  }
+
+  /**
+   * Clone an entry without status and stage (for Install without async).
+   * @return entry
+   */
+  @JsonIgnore
+  public TenantModuleDescriptor cloneWithoutStatus() {
```

Comment:
`status` has been renamed to `stage`.
Should the javadoc been renamed to `Clone an entry without stage` and the method been renamed to `cloneWithoutState`?

Reply:
I'll rename it again and update the information about the purpose of this function.